CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [2.1.4] - 2021-01-07
```

Comment:
2.1.5 ;)

Reply:
Whoops forgot lol, added the link for 2.1.4 as well since that was missing.